lib/chef/resource.rb

```diff
@@ -152,7 +152,7 @@ def action(arg = nil)
         arg.each do |action|
           validate(
             { action: action },
-            { action: { kind_of: Symbol, equal_to: allowed_actions } }
+            { action: { kind_of: Symbol, equal_to: allowed_actions.to_h.keys } }
```

Comment:
All of these `to_h` will be made better once I update `allowed_actions` in my next push.